lib/codit/population/networks/city.py

```diff
@@ -79,37 +139,59 @@ def build_class_groups(people):
     logging.info(f"Only putting children >{MINIMUM_CLASS_AGE} years old into classrooms.")
     return classrooms
 
-
-def build_households(people):
+def build_households_home_list(total_h=50000):
```

Comment:
This is a bug chunk of new code - I wonder if it belongs in a different module?

Reply:
Agree,  I will move this build_households_home_list() to home.py